content/docs/conditional-rendering.md

```diff
@@ -150,7 +150,7 @@ Bu kod JavaScript'te çalışır çünkü `true && expression` her zaman `expres
 
 Bu yüzden, eğer koşulunuz `true` ise, `&&`'den sonra yazacaklarınız çıktı olur. Eğer koşulunuz `false` ise, React onu görmezden gelip atlayacaktır.
 
-Değer olarak `falsy` bir ifade döndürmek, React'in `&&` ifadesinden sonra gelen kısmı atlamasına neden olacaktır. Ancak yine de o `falsy` değeri dönüp render edeceğini unutmayın. Aşağıdaki örnekte `render` metodundan `<div>0</div>` değeri dönecektir:
+Değer olarak `false` bir ifade döndürmek, React'in `&&` ifadesinden sonra gelen kısmı atlamasına neden olacaktır. Ancak yine de o `falsy` değeri dönüp render edeceğini unutmayın. Aşağıdaki örnekte `render` metodundan `<div>0</div>` değeri dönecektir:
```

Comment:
Actually, this should be `falsy` not `false`. `Falsy` means any expression that has a boolean value of `false` when evaluated:

```js
undefined && 1 
```
The result for the above is `undefined` when evalulated. It is not `false` but `falsy`. So we should leave the statement as it is.